src/OrchardCore.Modules/OrchardCore.Resources/ResourceManifest.cs

```diff
@@ -121,6 +121,12 @@ public void BuildManifests(IResourceManifestBuilder builder)
                 .SetCdnIntegrity("sha384-STc8Gazx86A+NmeBWQTqa5Ob1wGSRQZevexYiUkKdiqZhi5LSZ28XYAvgptHK5HH")
                 .SetVersion("5.0.12")
                 ;
+
+            manifest
```

Comment:
If it's supposed to be reused by many themes, which fine, then make it a reusable script where we can change the target element to search, and change the class to add.